### PR TITLE
Feat/wallet connect

### PR DESCRIPTION
**Pull request by gabrielbazan7**
needs https://github.com/bitpay/bitcore/pull/2984

BitPay (2)

BitPay (3)

BitPay (1)

### CODE REVIEW

**Comment by Gamboster** [comment]
I tried it and worked fine.
Good job improving the UI!

File: src/pages/integrations/wallet-connect/wallet-connect.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                <div class="item" *ngIf="param.to">
+                  <div>{{ 'To' | translate }}</div>
+                  <ion-note class="ellipsis" item-end>
+                    {{ param.to }}
+                  </ion-note>
+                </div>
+                <div class="item" *ngIf="param.gas">
+                  <div>{{ 'Gas Limit' | translate }}</div>
+                  <ion-note item-end>
+                    {{ param.gas }}
```
Comment (Gamboster) [inline src/pages/integrations/wallet-connect/wallet-connect.html:121]: Should we use `satToUnit` pipe with Gas, Gas limit or Gas Price?

Comment (gabrielbazan7) [reply src/pages/integrations/wallet-connect/wallet-connect.html:121]: no with Gas Limit

**Review by Gamboster — DISMISSED** [review]
Great job. Very excited for this feature!

**Review by cmgustavo — DISMISSED** [review]
Great work! 💯

**Review by cmgustavo — APPROVED** [review]
🙌

**Pull request merged**